python/llm/test/benchmark/check_results.py

```diff
@@ -20,26 +20,44 @@
 import sys
 import argparse
 import pandas as pd
+from omegaconf import OmegaConf
 
 def main():
     parser = argparse.ArgumentParser(description="check if the number of lines in html meets expectation")
-    parser.add_argument("-n", "--expected_lines", type=int, dest="expected_lines",
-                        help="the number of expected html lines", default=0)
```

Comment:
we can keep -n
The logic may be:
- if -y exists, use -y
- else if -n exists, use -n

Reply:
ok

Reply:
sure